guides/common/modules/proc_configuring-dhcpd-to-use-with-server.adoc

```diff
@@ -19,22 +12,36 @@ This is required because {Project} creates configuration files on the TFTP serve
 If the `dhcp-no-override` setting is disabled, hosts fetch the boot loader and its configuration from the root directory, which might cause an error.
```

Comment:
This whole paragraph doesn't make sense to me. AFAIK the external DHCP server doesn't support dnsmasq. It was added in 006117785d7dc02a734cf9beab3912ab21f1b7bc but if I read back https://bugzilla.redhat.com/show_bug.cgi?id=1841101 I still have a hard time understanding this.

You can use dnsmasq as a DHCP server, but we would describe that as Unmanaged DHCP which we're not documenting at all in the current guide.

My recommendation is to drop this note and track figuring out unmanaged DHCP as a follow up.

Reply:
I removed the note.